### DISCUSSION TITLE
How do we use link relative functions | need help please

### DISCUSSION THREAD

**Discussion by Cruising0904** [nvim-telekasten/telekasten.nvim#224] [Q&A] [answered]
After few days of asking how I set directory the other day, now I was going to create new notes start using Telekasten. But I bumped into few things

1. It is quite different than I know(Github markdown), Insert link brings two squrebrackets indside of file relative path. but it does not show link on preview page.

2. follow link does not let me go to the link - when I press Enter key in picker nothing happens. Is there other key combination to go to the file? 

3. I am going to organize directory by categories, if I trigger new note then I have to move it to the directory manually? I find that / <- does not work.. 

Any helps apprecited Thanks!

**Comment by lambtho12** [comment, marked as answer]
I suggest you read the documentation on links. A [short summary is detailed in the repo]([notation](https://github.com/renerocksai/telekasten.nvim#link-notation)), more info are given in the vim documentation when you type `:h telekasten`.

To reply to your questions:
1. Telekasten links are always what we call `wiki` style: `[[mynote]]`. Regular markdown links are reserved for media (images): `[image](myimage.png)`. This is currently a limitation that we are well aware of (see #126 for instance). We plan on allowing multiple types of links, but this is not something we are currently working on right now. I do not quite get the `it does not show link on preview page`. My guess is that the link is malformed or the file does not exist.
2. There is probably a configuration error here (or your link is malformed). Normally you should place your cursor over the link and call `follow_link()`; then it will open a picker with the link pre-filled. You just need to hit Enter to open the file in a new buffer.
3. Subdirectories are supported. You may want to use the `subdirs_in_links = true` option in your telekasten configuration. When you do that, you can use the New note command and type the title as `subdir/mynote`. It will automatically place the note in the proper subdir. Note that at the moment it does not really create the directory if it does not exist on your system (I am working on it, but I have little time, so progress is slow). Make sure the subdirectory are created first then!

**Reply by Cruising0904** [reply]
:h telekasten shows not much different than wiki page, I already read through wiki page. 
For 1, 2 you are right. It only works with [[relative path]] 
thanks for answer! cheers :+1:

**Reply by lambtho12** [reply]
Your links were `[[absolute_path]]` beforehand?
We could probably add a feature request for that to be honest... Ideally I would like to refactor all path creation and manipulation functions in telekasten in order to have a more streamlined process.
